edgelet/contrib/config/linux/debian/config.yaml

```diff
@@ -85,13 +100,19 @@ provisioning:
 #     device_ca_pk     - path to the device ca private key file
 #     trusted_ca_certs - path to a file containing all the trusted CA
 #                        certificates required for Edge module communication
-#
+# Note:
+#     The values for these fields could be specified as a file path or a URI. 
+#     URI is preferred as it provides flexibility in where and 
+#     how these resources are to be accessed.
+#     Ex. when specifying a PEM encoded private key file, the URI
+#     could be specified as "file:///path/cert_key.pem" or 
+#     "/path/cert_key.pem"
 ###############################################################################
 
 # certificates:
-#   device_ca_cert: "<ADD PATH TO DEVICE CA CERTIFICATE HERE>"
-#   device_ca_pk: "<ADD PATH TO DEVICE CA PRIVATE KEY HERE>"
-#   trusted_ca_certs: "<ADD PATH TO TRUSTED CA CERTIFICATES HERE>"
+#   device_ca_cert: "<ADD PATH OR URI TO DEVICE CA CERTIFICATE HERE>"
```

Comment:
Let me know if this reads better.

Reply:
The funny thing is that registration id for TPM is also optional but since GA this was made as a required field. To make this optional for the TPM is not breaking but will need additional changes to support this.

Reply:
Perhaps this was required for the prior DPS API before and is optional now. Regardless, we can address making this optional for TPM flow as a separate task.